### PR TITLE
DHFPROD-5148: Ability to show multiple values in a table view for a property.

### PR DESCRIPTION
**Pull request by timur-isangulov**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/components/queries/queries.tsx [hunk 1]
```diff
@@ -202,12 +202,14 @@ const Query = (props) => {
             toggleSaveChangesIcon(false);
             toggleDiscardIcon(false);
         }
+
+        // setColumnSelectorTouched(JSON.stringify(searchOptions.selectedTableProperties) !== JSON.stringify(props.columns))
```
Comment (bsrikan) [inline marklogic-data-hub-central/ui/src/components/queries/queries.tsx:206]: Can we delete these commented out lines.

File: marklogic-data-hub-central/ui/src/components/queries/queries.tsx [hunk 2]
```diff
@@ -65,7 +65,8 @@ const Query = (props) => {
                     entityTypeIds: searchOptions.entityTypeIds.length ? searchOptions.entityTypeIds : props.entities,
                     selectedFacets: facets,
                 },
-                propertiesToDisplay: props.columns,
+                // propertiesToDisplay: props.columns,
```
Comment (bsrikan) [inline marklogic-data-hub-central/ui/src/components/queries/queries.tsx:68]: Can we delete these commented out lines.

File: marklogic-data-hub-central/ui/src/components/results-tabular-view/results-tabular-view.test.tsx [hunk 3]
```diff
@@ -47,4 +46,23 @@ describe("Results Table view component", () => {
...
+                />
+        )
+        // Check table column headers are rendered
+        expect(queryByText('Carmdin')).toBeNull();
+        expect(queryByText('Carm din')).toBeNull();
+        expect(getByText('Carm')).toBeInTheDocument();
+        expect(getByText('din')).toBeInTheDocument();
+        expect(getByText('Carm')).toContainHTML('<div style="text-overflow: ellipsis; overflow: hidden;">Carm</div>');
+        expect(getByText('din')).toContainHTML('<div style="text-overflow: ellipsis; overflow: hidden;">din</div>');
+    });
```
Comment (bsrikan) [inline marklogic-data-hub-central/ui/src/components/results-tabular-view/results-tabular-view.test.tsx:67]: both "Carm" and "din" can be in different cells and still the test would pass. We want to verify that both Carm and din are in the same cell. Try using:
expect(getByText('Carm').closest('td')).toEqual(getByText('din').closest('td'))

File: marklogic-data-hub-central/ui/src/components/queries/queries.tsx (hunk 1 above)
Comment (timur-isangulov) [reply marklogic-data-hub-central/ui/src/components/queries/queries.tsx:206]: Removed.

File: marklogic-data-hub-central/ui/src/components/queries/queries.tsx (hunk 2 above)
Comment (timur-isangulov) [reply marklogic-data-hub-central/ui/src/components/queries/queries.tsx:68]: Removed.

File: marklogic-data-hub-central/ui/src/components/results-tabular-view/results-tabular-view.test.tsx (hunk 3 above)
Comment (timur-isangulov) [reply marklogic-data-hub-central/ui/src/components/results-tabular-view/results-tabular-view.test.tsx:68]: Added. Thank you.

**Pull request merged**